app/src/main/res/values/strings.xml

```diff
@@ -810,4 +810,6 @@ Upload your first media by tapping on the add button.</string>
     <item quantity="one">%d image selected</item>
     <item quantity="other">%d images selected</item>
   </plurals>
+  <string name="multiple_files_depiction">Please remember that all images in a multi-upload should belong to the same category/depiction. If the images do not share depictions and categories, please perform several uploads.</string>
+  <string name="multiple_files_depiction_header">Note about multi-uploads</string>
```

Comment:
Would you mind adding a "Do not show again" checkbox on that dialog?
Thanks a lot and sorry for the delay! :-)